### ISSUE TITLE
erroneous "deprecated-inline-view-helper" violation

### ISSUE THREAD

**Issue by csantero** [ember-template-lint/ember-template-lint#191]
With 0.6.4, the `deprecated-inline-view-helper` rule is triggering for the following syntax:

```
{{foo-bar qux="view"}}
```

Yielding the linter message "The inline form of `view` is deprecated. Please use the `Ember.Component` instead."

**Comment by rwjblue** [comment]
Should be fixed by https://github.com/ember-template-lint/ember-template-lint/pull/431 when it lands...

**Comment by boris-petrov** [comment]
Still happening with version 3.9.0. :)

**Comment by bmish** [comment]
Looks like we'll need a volunteer to continue working on the fix that was started in #431...

**Comment by lifeart** [comment]
closing as likely outdated and there is no active champions on it

**Comment by boris-petrov** [comment]
@lifeart why did you close this? It's still an issue in the current version - 7.0.8.

**Comment by lifeart** [comment]
@boris-petrov issue remains as-is for 3 years and related PR is closed, and nobody working on it, I'm happy to reopen it if we found contributor to fix it.
